Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -70,6 +97,22 @@ class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > :
         }
     }
     
+    private func interpolate(route: Route?) -> [CLLocationCoordinate2D?] {
+        guard let route = route else { return [] }
+        
+        var distanceAway: CLLocationDistance = 0
+        let distance = route.distance/route.expectedTravelTime
```

Comment:
This could result in division by zero if the route’s waypoints happen to snap to the same coordinate.

Reply:
oh my..I will fix this then.